### PR TITLE
DOCS-8448 Java Web Tokens

### PR DESCRIPTION
**Pull request by wwarner-salesforce**

### CODE REVIEW

File: modules/ROOT/pages/data-bridge.adoc [hunk 1]
```diff
@@ -8,133 +8,98 @@ The data bridge is the secure connection from Anypoint API Community Manager to
...
+. Set *Audiences* to `+https://anypoint.mulesoft.com/accounts/api/v2/oauth2/token+`.
+. Set *JWT Signing Certificate* to the certificate you downloaded when you created a key pair in the previous steps.
+. In the *Callout Options* section, enable *Generate Authorization Header*.
 
-NOTE: Remember to associate the external data source with this authentication provider.
+== Modify External Data Source
 
-== Authenticate the Data Bridge [[authenticate-data-bridge]]
+. Click *Setup* > *External Data Source* > *Exchange*.
+. Set *URL* to `callout:acm_pkg__{anypoint_mulesoft/odatabridge/odata.svc/`.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/data-bridge.adoc]: Remove {

**Review by ferferrarazzo — APPROVED** [review]

**Review by jdeoliveira — DISMISSED** [review]

File: modules/ROOT/pages/data-bridge.adoc [hunk 2]
```diff
@@ -8,133 +8,122 @@ The data bridge is the secure connection from Anypoint API Community Manager to
...
-== Create an Authentication Provider in Your Salesforce Organization [[create-auth-provider]]
+== Modify External Data Source
 
-In the following task steps, if your Anypoint Platform organization is hosted in the EU, replace all instances of `+https://anypoint.mulesoft.com+` with `+https://eu1.anypoint.mulesoft.com+`.
+. Click *Setup* > *External Data Source* > *Exchange*.
+. Set *URL* to `callout:acm_pkg__anypoint_mulesoft/odatabridge/odata.svc/`.
+. Go to the *Authentication* section.
+. Set *Identity* to *Anonymous*.
+. Set *Authentication Protocol* to *No Authentication Protocol*.
+. Click *Setup* > *External Data Source* > *Exchange* > *Custom HTTP Headers*.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/data-bridge.adoc]:
```suggestion
. Click *Save*
```

Comment (ferferrarazzo) [reply modules/ROOT/pages/data-bridge.adoc]: In this section after save the external data source the user is already incustom http headers

File: modules/ROOT/pages/data-bridge.adoc [hunk 3]
```diff
@@ -8,133 +8,122 @@ The data bridge is the secure connection from Anypoint API Community Manager to
...
+== User Configuration
 
-NOTE: Remember to associate the external data source with this authentication provider.
+Give your user the *Organization Administrators* role:
 
-== Authenticate the Data Bridge [[authenticate-data-bridge]]
+. In your Anypoint Platform organization, click *Access Management* > *Users*.
+. Select your user.
+. Click *Role* > *Add role by name*.
+. Add the role *Organization Administrators*.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/data-bridge.adoc]:
```suggestion
. Add the role *Exchange Contributor*.
```

**Review by ferferrarazzo — CHANGES_REQUESTED** [review]

**Review by ferferrarazzo — DISMISSED** [review]

**Review by ferferrarazzo — APPROVED** [review]

**Comment by wwarner-salesforce** [comment]
Moved to https://github.com/mulesoft/docs-api-community-manager/pull/161 to facilitate later editorial review. Thank you, everyone!

**Pull request closed without merging**